RedHatInsights/insights-inventory-frontend
# feat(ESSNTL-4491): Sync URLSearchParams with the table filters

Implements https://issues.redhat.com/browse/ESSNTL-4491.

This PR makes it possible to preset the groups table (/groups) filters, pagination and sorting by providing appropriate query parameters with URL.

## How to test

1. Go to /groups
2. Try to apply different filters, sorting, pagination
3. Reload the page and make sure the same settings applied
